src/legacy/core_plugins/embeddable_api/public/panel/panel_header/panel_header.tsx

```diff
@@ -78,7 +102,8 @@ function PanelHeaderUi({
           }
         )}
       >
-        {hidePanelTitles ? '' : title}
+        {showTitle ? title : ''}
+        {renderBadges(badges, embeddable)}
```

Comment:
We need a space between the panel title and the badge. 
<img src="https://d.pr/free/i/VFkkhY+" width="50%" />

Can you just add a `&nbsp;` after the title, **if** `showTitle`?